packages/desktop/views/dashboard/governance/components/ProposalCard.svelte

```diff
@@ -43,7 +46,16 @@
         <Text truncate>{proposal.title}</Text>
     </div>
     <div class="flex justify-between items-center">
-        <ProposalStatusInfo {proposal} />
+        <div class="flex items-center gap-2">
+            <ProposalStatusInfo {proposal} />
+            {#if proposal?.status === EventStatus.Holding}
+                {@const remainingTime = getTimeDifference(
+                    milestoneToDate($networkStatus.currentMilestone, proposal?.milestones?.ended),
+                    $time
+                )}
+                <Pill color="neutral">{remainingTime}</Pill>
+            {/if}
```

Comment:
`remainingTime` can be undefined
```suggestion
            {#if proposal?.status === EventStatus.Holding}
                {@const remainingTime = getTimeDifference(
                    milestoneToDate($networkStatus.currentMilestone, proposal?.milestones?.ended),
                    $time
                )}
                {#if remainingTime}
                    <Pill color="neutral">{remainingTime}</Pill>
                {/if}
            {/if}
```